### ISSUE TITLE
Programmatic cluster addition to Lens

### ISSUE THREAD

**Issue by povils** [lensapp/lens#1070]
**What would you like to be added**:
A programmatic and unified way to modify files like `lens-cluster-store.json` in order to add clusters

**Why is this needed**:
It is manageable to add clusters when they are long-lived and there are few of them. However, when cluster number is double digit and they are short-lived it becomes cumbersome to make my Lens data up-to-date. I could myself write some script which gets all the clusters from "cluster registry" and then writes the info into Lens related files but it would be awesome that Lens could provide some API compatability through some script: lensctl ? :D . My usecase would be satisfied with such functionality/API:
```
lensctl add  {{cluster name}}  --kubeconfig={{path to kubeconfig}} --workspace=production
```

**Environment you are Lens application on:**
3.6.7

**Comment by matti** [comment]
Yes, also great for #1072 https://github.com/supervisor-com/lens-web

**Comment by jakolehm** [comment]
With Lens 5.0.0-beta.6 it's already possible to add clusters by writing kubeconfigs to `~/.kube/` folder (Lens autodetects those files).

**Comment by Nokel81** [comment]
Fixed by #2567 (and subsequent PRs)
